src/__tests__/definitions/valid-task-intrisic-function.json

```diff
@@ -1,28 +0,0 @@
-{
-  "Comment": "Different resource types for tasks.",
-  "StartAt": "GetAttResource",
-  "States": {
-    "GetAttResource": {
-      "Type": "Task",
-      "Resource": {
-        "Fn::GetAtt": [
-          "lambda-name_GetAtt",
-          "Arn"
-        ]
-      },
-      "Next": "RefResource"
-    },
-    "RefResource": {
-      "Type": "Task",
-      "Resource": {
-        "Ref": "lambda-name_Ref"
-      },
```

Comment:
@ChristopheBougere this looks like a deployment extension. This could be supported if needed.